core/src/main/java/org/apache/accumulo/core/clientImpl/bulk/BulkImport.java

```diff
@@ -127,9 +131,13 @@ public void load()
 
     SortedMap<KeyExtent,Bulk.Files> mappings;
     TableOperationsImpl tableOps = new TableOperationsImpl(context);
+    Retry retry = Retry.builder().infiniteRetries().retryAfter(100, MILLISECONDS)
+        .incrementBy(100, MILLISECONDS).maxWait(2, SECONDS).backOffFactor(1.5)
```

Comment:
I would set the max to a minute or two.

Reply:
I don't see a way to set a max total time with Retry... we either have to use ```infiniteRetries()``` or set a number of retries with ```maxRetries(long max)```

Reply:
I was thinking of setting `maxWait(` higher

Reply:
Ah I see.  Yeah that would be better